packages/client/webapp/src/components/Header.vue

```diff
@@ -1,33 +1,69 @@
 <script setup lang="ts">
-import { computed } from 'vue';
+import Button from '@/components/Button.vue';
 import IconLogin16 from '@carbon/icons-vue/es/login/16';
+import IconSearchLocate16 from '@carbon/icons-vue/es/search--locate/16';
+import { useRouter } from 'vue-router';
 import { useAuthStore } from '../stores/auth';
 
 const auth = useAuthStore();
-const isAuthenticated = computed(() => auth.isAuthenticated);
+const router = useRouter();
 
 const login = () => window.location.assign('http://localhost:8078');
+const homepage = () => router.push('/');
+
+const projectName = 'Name of the project that can be long for clarity and precision';
 </script>
 
 <template>
 	<nav>
-		<!-- Use relative path instead of alias for now as Component Tests seem to have an issue -->
-		<img src="@assets/images/logo.png" height="32" width="128" alt="logo" />
-		{{ auth.name }}
-		<Button v-if="!isAuthenticated" @click="login">Login <IconLogin16 /></Button>
+		<header>
+			<img src="@assets/images/logo.png" height="32" width="128" alt="logo" />
+			<p>
+				<a @click="homepage">Projects</a>
+				<span>{{ projectName }}</span>
+			</p>
+		</header>
+
+		<Button class="dataExplorer" @click="router.push('/explorer')"><IconSearchLocate16 /></Button>
+		<Button v-if="!auth.isAuthenticated" @click="login">Login <IconLogin16 /></Button>
 	</nav>
 </template>
 
 <style scoped>
 nav {
 	align-items: center;
+	background-color: var(--un-color-body-surface-primary);
 	border-bottom: 1px solid var(--un-color-black-20);
```

Comment:
If we are using box shadows I think we should get rid of the solid borders. The header border makes the mode config almost look like its 1px higher than the sidebar nav. Since you changed the sidebar nav to a darker green I think it looks cleaner without its solid border as well.

Current version:
<img width="266" alt="image" src="https://user-images.githubusercontent.com/28237258/195202410-23455cc9-1cde-40f5-9d09-777e18b8b7e1.png">

Without solid borders:
<img width="261" alt="image" src="https://user-images.githubusercontent.com/28237258/195202475-e6ac8013-e3fc-4264-b8df-2eb9da12759d.png">

Reply:
Thank you! I think they got back from a merge from `main`.